obs-check-devel-openqa-leap-extra-dependencies

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -e
+
+osc="osc --apiurl https://api.opensuse.org"
+
+for project in $($osc search --project -s 'devel:openQA:Leap:' | grep -o '^devel:openQA:Leap:.*'); do
+    for package in $($osc list "$project"); do
+        $osc api "/comments/package/$project/$package" | \
+            yq -pxml -oy '.comments.comment |= ([] + .) | .comments.comment[].+content' | \
+            grep -q "Reason for linking:" || \
+            { echo "No reason for $project/$package" >&2; problem=1; }
+    done
+done
+
+[ -n "$problem" ] && exit 1
+exit 0
```

Comment:
That's equivalent to just

```
test -z "$problem"
```

isn't it?

Reply:
yeah, right, now it is 😄